packages/quantic/cypress/e2e/facets-1/facet/facet.cypress.ts

```diff
@@ -787,7 +787,14 @@ describe('Facet Test Suite', () => {
       });
 
       describe('with invalid sorting', () => {
-        before(() => {
+        beforeEach(() => {
+          // This error occasionally occurs with the Salesforce Lightning Modal component, although we don't know exactly why it occurs, we do know that it only occurs in this Cypress test environment and never in the production environment.
+          cy.on('uncaught:exception', (err) => {
```

Comment:
Could this be refactored into a common place? If only certain tests need to have this then I guess this is better than having the common handler check for test names etc?

> To conditionally turn off uncaught exception handling for a certain error
A great place to put this configuration is in the [supportFile](https://docs.cypress.io/guides/core-concepts/writing-and-organizing-tests#Support-file), since it is loaded before any test files are evaluated.

Reply:
> we do know that it only occurs in this Cypress test environment and never in the production environment. 

Is this based on production logs/monitoring or just that it hasn't been reported in prod?